src/web/components/SignInGate/gates/fake-social-variant.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Suspense } from 'react';
+import { Lazy } from '@root/src/web/components/Lazy';
+
+import { SignInGateComponent } from '@frontend/web/components/SignInGate/gateDesigns/types';
+import { canShow } from '@frontend/web/components/SignInGate/displayRule';
+import { initPerf } from '@root/src/web/browser/initPerf';
+
+const SignInGateFakeSocial = React.lazy(() => {
+	const { start, end } = initPerf('SignInGateFakeSocial');
+	start();
+	return import(
+		/* webpackChunkName: "SignInGateFakeSocial" */ '../gateDesigns/SignInGateFakeSocial'
+	).then((module) => {
+		end();
+		return { default: module.SignInGateFakeSocial };
+	});
+});
+
+export const signInGateComponent: SignInGateComponent = {
```

Comment:
I'm far from sure about this but is this component exporting tsx or can it just use `.ts`? If it _is_ tsx, should it use the PascalCase naming convention? I feel like React normally enforces this

I'm super unsure about all of this though so feel free to ignore me!

Reply:
ah interesting! i think, i have a feeling that we didnt use pascal case due to the type being `SignInGateComponent` but I guess we could refactor it to be something line `FakeSocialSignInGateComponent` it's out of scope for this PR, but i'll keep it in mind